android/src/main/kotlin/com/gunschu/jitsi_meet/JitsiMeetPluginActivity.kt

```diff
@@ -26,27 +29,33 @@ class JitsiMeetPluginActivity: JitsiMeetActivity() {
 
     var onStopCalled: Boolean = false;
 
-    override fun onPictureInPictureModeChanged( isInPictureInPictureMode: Boolean, newConfig: Configuration?)
-    {
+    override fun onPictureInPictureModeChanged(isInPictureInPictureMode: Boolean, newConfig: Configuration?) {
         super.onPictureInPictureModeChanged(isInPictureInPictureMode, newConfig)
 
-        if (isInPictureInPictureMode == false && onStopCalled)
-        {
+        if (isInPictureInPictureMode == false && onStopCalled) {
             // Picture-in-Picture mode has been closed, we can (should !) end the call
             getJitsiView().leave()
         }
     }
 
-    override fun onStop()
-    {
+    private val myReceiver: BroadcastReceiver = object : BroadcastReceiver() {
+        override fun onReceive(context: Context, intent: Intent) {
+            when (intent?.action) {
+                JITSI_MEETING_CLOSE -> finish()
```

Comment:
Have you tried getJitsiView().leave() as above? Trying to keep consistency as well as maybe there is some clean up that leave() call does we don't want to miss out on.